server/iframe.go

```diff
@@ -222,14 +261,98 @@ func (a *API) authenticate(w http.ResponseWriter, r *http.Request) {
 	http.SetCookie(w, csrfCookie)
 
 	// Redirect to the home page
-	http.Redirect(w, r, "/", http.StatusSeeOther)
+	http.Redirect(w, r, redirectPath, http.StatusSeeOther)
+}
+
+// MessageHasBeenPosted is called when a message is posted in Mattermost. We rely on it to send a user activity notification
+// to Microsoft Teams when a user is mentioned in a message.
+// This is called in a controller Goroutine in the server side so there's no need to worry about concurrency here.
+func (p *Plugin) MessageHasBeenPosted(c *plugin.Context, post *model.Post) {
+	p.API.LogError("Message has been posted", "post_id", post.Id, "post_participants", post.Participants)
```

Comment:
I'm assuming this was for debugging, but definitely don't want to log on each call of this method.